tests/lightning_qubit/test_measurements_class.py

```diff
@@ -819,7 +818,6 @@ def test_controlled_qubit_unitary_from_op(self, tol, lightning_sv):
 
         assert np.allclose(result, expected, tol)
 
-    @flaky(max_runs=5)
```

Comment:
No shots, so after fixing initial state with seed, this should be deterministic.